assets/javascripts/discourse/components/chat-live-pane.js

```diff
@@ -526,6 +536,14 @@ export default Component.extend({
       this._scrollGithubOneboxes();
       this._pluginsDecorators();
       this._highlightCode();
+
+      document
+        .querySelectorAll(".chat-message-text")
```

Comment:
This resolves my question about passing the whole `this.element`. 🙂

Reply:
ok I see ;) I guess we agree then :D